Code/Framework/AzFramework/Platform/Common/Xcb/AzFramework/XcbNativeWindow.h

```diff
@@ -55,9 +55,14 @@ namespace AzFramework
         void InitializeAtoms();
         void GetWMStates();
 
+#ifdef SPLASH_IMAGE_EXISTS
```

Comment:
I would left draw splash function. I would #ifdef SPLASH_IMAGE_EXISTS only used inside DrawSplash implementation. We should use as little macros as possible.

Reply:
But on the other hand this leaves a function that is unimplemented, which isn't much better